hw6/src/main/java/ru/otus/hw6/commands/BookCommands.java

```diff
@@ -0,0 +1,52 @@
+package ru.otus.hw6.commands;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.shell.standard.ShellComponent;
+import org.springframework.shell.standard.ShellMethod;
+import ru.otus.hw6.converters.BookConverter;
+import ru.otus.hw6.services.BookService;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@ShellComponent
+public class BookCommands {
+
+    private final BookService bookService;
+
+    private final BookConverter bookConverter;
+
+    @ShellMethod(value = "Find all books", key = "ab")
+    public String findAllBooks() {
+        return bookService.findAll().stream()
+                .map(bookConverter::bookToString)
+                .collect(Collectors.joining("," + System.lineSeparator()));
+    }
+
+    @ShellMethod(value = "Find book by id", key = "bbid")
+    public String findBookById(long id) {
+        return bookService.findById(id)
+                .map(bookConverter::bookToString)
+                .orElse("Book with id %d not found".formatted(id));
+    }

```

Comment:
shell:>bbid 1
failed to lazily initialize a collection of role: ru.otus.hw6.models.Book.genres: could not initialize proxy - no Session